# goal
sensor_msgs/Image image
int64 number_of_faces
mdr_perception_msgs/FaceBoundingBox[] bounding_boxes
---
# result
string[] genders
bool success
---
# feedback
string current_state
string message

